spec/definitions.yaml

```diff
@@ -181,9 +181,10 @@ definitions:
           to use in the new cluster
       availability_zones:
         description: |
-          Number of availability zones a cluster should be spread across. The
-          default is provided via the [info](#operation/getInfo) endpoint.
-        type: integer
+          AWS - Stringified number of availability zones a cluster should be spread across.
+          Azure - A serialized list of availability zones a cluster should be spread across
+          The default is provided via the [info](#operation/getInfo) endpoint.
+        type: string
```

Comment:
This is a breaking change

Reply:
Depends on where you define the breaking change line. This would be a breaking change for clients that implement the generated go library like gsctl, I suppose, but only when they update that library.

Any existing code or client would still interface correctly, whether it sends a `"3"` or a `3`

So no existing implementations would break. That's what we were thinking of when we thought "not a breaking change".

Anyways just being the devil's advocate here, because I'd prefer not to go for this approach. I'm personally in favour of something like `azure.availability_zones` , because then it also gives us a top level place to add more azure specific things if needed.